src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -977,7 +977,7 @@ uint32 Unit::DealDamage(Unit* attacker, Unit* victim, uint32 damage, CleanDamage
 
         //if (attacker && victim->GetTypeId() == TYPEID_PLAYER && victim != attacker)
         //victim->ToPlayer()->UpdateAchievementCriteria(ACHIEVEMENT_CRITERIA_TYPE_TOTAL_DAMAGE_RECEIVED, health); // pussywizard: optimization
-
+        std::this_thread::sleep_for(400ms);
```

Comment:
Add a little comment as the other PR 👍 
All good

Reply:
added